packages/cloning-decorator/.eslintrc.json

```
@@ -1,6 +1,19 @@
 {
   "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*", "node_modules/"],
   "overrides": [
```

Remove the rule from the override since it is included in the base file

```suggestion
{
  "extends": ["../../.eslintrc.json"],
  "overrides": [
    {
      "files": ["*.json"],
      "parser": "jsonc-eslint-parser",
      "rules": {}
    }
  ]
}
```